pkg/handler/publicHandler.go

```diff
@@ -1487,11 +1490,13 @@ func (h *PublicHandler) ExecuteConnector(ctx context.Context, req *connectorPB.E
 	resp = &connectorPB.ExecuteConnectorResponse{}
 	connID, err := resource.GetRscNameID(req.GetName())
 	if err != nil {
+		span.SetStatus(1, err.Error())
 		return resp, err
 	}
 
 	owner, err := resource.GetOwner(ctx, h.service.GetMgmtPrivateServiceClient())
 	if err != nil {
+		span.SetStatus(1, err.Error())
 		return resp, err
 	}
 	connector, err := h.service.GetConnectorByID(ctx, connID, owner, true)
```

Comment:
Since we share the `connector` now, so the `isBasicView` need to be `false`

Reply:
ok, let me update it